README-parallels.md

```diff
@@ -23,3 +23,8 @@ Host setup or installations
  9.`vagrant up`
 
 The above setup was verified on macOS Sonoma on Apple M3 Pro.
+
+## Supported Boxes
+
+We do not currently [test](tests/test-paralles.sh) the Parallels provider, it
```

Comment:
when I tested this branch successfully, I used `export VAGRANT_BOX=focal-m1`. I also couldn't find a Vagrant box named `focal-m1` anymore, so maybe it was using my local copy of it.

However I was able to find some other boxes (both focal and jammy) on Macbook:
https://portal.cloud.hashicorp.com/vagrant/discover/gvengel/salt-focal-arm64
https://portal.cloud.hashicorp.com/vagrant/discover/gvengel/salt-jammy-arm64

I think it's fine to have a follow-up to update those boxes for Parallels support (I can help with that), since I have a broken master branch already and this branch fixed issues for me.